source/Playnite/Localization.cs

```diff
@@ -127,6 +137,10 @@ public static void SetLanguage(string language)
                 }
 
                 dictionaries.Add(res);
+
+                CultureInfo culture = new CultureInfo(language.Replace("_", "-"), false);
+                CultureInfo.DefaultThreadCurrentCulture = culture;

```

Comment:
I'm not sure if it's a good idea to force application culture based on the app language. For example I personally use Playnite in English, but I also use Czech system culture and I want to have dates and times in Czech in Playnite. I think we should be preferring system culture. But please correct me if this is here for some other reason that I'm not aware of for Hebrew support specifically.

Also this will not work for default English language and system culture will be used (the if statement will not be called in that case).

Reply:
I personally use English system culture and hebrew dates are formatted differently from English.
But I don't mind have it how you described it. I will create a local CultueInfo object that will be used to query for culture properties (i.e RTL)